draft-ietf-modpod-group-processes.md

```diff
@@ -622,7 +622,12 @@ is non-exhaustive.
 - "Sealioning", where a participant makes incessant requests for evidence or
   data, even while remaining superficially polite.
 
-These items are just examples. The moderator team's task consists of
+These items are just examples. If a particular example of moderation cannot
+be matched to an example that appears in this list, this has no bearing at
+all on whether or not the moderation action was warranted. Its absence
+from this list therefore cannot serve as a basis for an appeal.
```

Comment:
I think this needs another shot, Ted.  Keep in view the text below.  I think you're just trying to say:

These are just examples.  That someone was disruptive in a way not listed here is not grounds for appeal.

Reply:
Yup, that's better. Brevity, soul of wit, etc.

Reply:
Ok, I will take this one on.  Let's leave this PR open and this comment unresolved for the next few days.

Reply:
We should not assume that because we all know the appendix is non-normative, that someone who gets moderated will accept that as an explanation for why their lawyering doesn't work.